### PR TITLE
Update logging in lambdas and run formatter

### PR DESCRIPTION
**Pull request by BarrettOlson**
Summary
---------

Updated lambdas to use common get_logger to avoid updating the root logger
and ran `black hasher-matcher-actioner/hmalib/`

Test Plan
---------
```
make upload_docker
terraform -chdir=terraform destroy
terraform -chdir=terraform apply
make dev_test_instance
```

### CODE REVIEW

File: hasher-matcher-actioner/hmalib/common.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import logging
+
+
+def get_logger(name=__name__, level=logging.INFO):
```
Comment (jeberl) [inline hasher-matcher-actioner/hmalib/common.py:6]: So `__name__` is reserved and refers to the name of what exactly? The calling class?

File: hasher-matcher-actioner/hmalib/lambdas/api/api_auth.py [hunk 2]
```diff
@@ -1,20 +1,16 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
-import logging
+
 import os
+from hmalib.common import get_logger
 
 ACCESS_TOKEN = os.environ["ACCESS_TOKEN"]
```
Comment (jeberl) [inline hasher-matcher-actioner/hmalib/lambdas/api/api_auth.py:6]: Is the plan to eventually store this as a secret in aws?

**Review by jeberl — APPROVED** [review]

File: hasher-matcher-actioner/hmalib/common.py (hunk 1 above)
Comment (schatten) [reply hasher-matcher-actioner/hmalib/common.py:6]: https://docs.python.org/3/reference/import.html?highlight=name#__name__

It's the fully qualified name of the module. eg. `hmalib.lambdas.yada.yada`.

File: hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_hasher.py [hunk 3]
```diff
@@ -43,7 +40,7 @@ def lambda_handler(event, context):
 
     1: https://docs.aws.amazon.com/lambda/latest/dg/images-create.html
     """
-
+    logger = get_logger(__name__)
```
Comment (schatten) [inline hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_hasher.py:43]: nit: have typically seen this done at the module level, not at the method.

File: hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py [hunk 4]
```diff
@@ -46,6 +42,15 @@ def get_index(bucket_name, key):
 
 
 def lambda_handler(event, context):
+    """
```
Comment (schatten) [inline hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py:45]: ❤️

**Review by schatten — APPROVED** [review]
LGTM. Thanks for linting all the files!

File: hasher-matcher-actioner/hmalib/lambdas/api/api_auth.py (hunk 2 above)
Comment (BarrettOlson) [reply hasher-matcher-actioner/hmalib/lambdas/api/api_auth.py:6]: Likely yes, I almost wrote that as a todo here. I am waiting to see what the frontend needs to have available before moving it though.

**Pull request merged**